src/components/visualizations/chart/highcharts/plugins/dualAxesLabelFormatter.ts

```diff
@@ -4,6 +4,42 @@ import head = require('lodash/head');
 import last = require('lodash/last');
 import { formatAsPercent, isInPercent } from '../customConfiguration';
 
+const DEFAULT_LIMIT_LENGTH = 4; // 0.00
```

Comment:
What does this comment mean?

Reply:
It means `'0.00'.length === 4`
Does it look confusing?

Reply:
I explained more in code comment :)